crc-admin-helper.spec

```diff
@@ -0,0 +1,82 @@
+# https://github.com/code-ready/admin-helper
+%global goipath         github.com/code-ready/admin-helper
+Version:                0.0.2
+
+%gometa
+
+# debuginfo is not supported on RHEL with Go packages
+%global debug_package %{nil}
+%global _enable_debug_package 0
+
+%global common_description %{expand:
+CodeReady Container's SUID helper}
+
+
+%global golicenses    LICENSE
+%global godocs        *.md
+
+Name:           %{goname}
+Release:        1%{?dist}
+Summary:        CodeReady Container's SUID helper
```

Comment:
SUID on linux, but administrator on Windows. Let's be consistent and use administrative privileges helper?

Reply:
Yup, I can change it, the summary won't be visible on Windows anyway